test/check-application

```diff
@@ -15,37 +15,64 @@ import testlib
 
 class TestApplication(testlib.MachineCase):
     def testBasic(self):
-        b = self.browser
-        m = self.machine
+        """
+        Tests if changing the language works as expected.
+        """
+        # Given on a cockpit instance
+        browser = self.browser
+        machine = self.machine
 
         self.login_and_go("/starter-kit")
+
+        self.goToStarterKit(browser, machine)
+
+        # When I change the language to German
+        browser.switch_to_top()
+        browser.click("#content-user-name")
+        browser.click(".display-language-menu a")
+        browser.wait_popup('display-language')
+        browser.set_val("#display-language select", "de-de")
+        browser.click("#display-language-select-button")
+        browser.expect_load()
+        # HACK: work around language switching in Chrome not working in current session (Cockpit issue #8160)
+        browser.reload(ignore_cache=True)
+        browser.wait_present("#content")
+
+        # Then the menu label (from manifest) should be translated
+        browser.wait_text("#host-apps a[href='/starter-kit']", "Bausatz")
+
+        browser.go("/starter-kit")
+        browser.enter_page("/starter-kit")
+        # And the page label (from js) should be translated
+        browser.wait_in_text(".container-fluid p", "Läuft auf")
+
+    def testChangingHostnameFileWatching(self):
+        # Given on a cockpit instance
+        browser = self.browser
+        machine = self.machine
+        initial_hostname = machine.execute("cat /etc/hostname").strip()
+
+        self.login_and_go("/starter-kit")
```

Comment:
This would make sense as part of `goToStarterKit()`, as it's the *only* piece of code that actually "goes" somewhere.